conda/recipes/rmm/meta.yaml

```diff
@@ -40,22 +41,36 @@ requirements:
     - ninja
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
-    - {{ compiler('cuda') }} {{ cuda_version }}
+    {% if cuda_major == "11" %}
+    - {{ compiler('cuda11') }} {{ cuda_version }}
+    {% else %}
+    - cuda-version {{ cuda_version }}
+    - {{ compiler('cuda') }}
+    {% endif %}
     - sysroot_{{ target_platform }} {{ sysroot_version }}
   host:
-    - cuda-python >=11.7.1,<12.0
+    {% if cuda_major == "11" %}
     - cudatoolkit ={{ cuda_version }}
+    - cuda-python >=11.7.1,<12.0a0
+    {% else %}
+    - cuda-python >=12.0,<13.0a0
```

Comment:
This is a good question -- but maybe we should defer this to a later PR, since it changes the CUDA 11 pinning strategy. I'll leave this open for now.

Reply:
This also led me down the path of investigating our `cuda-python` dependency. It turns out we should be able to drop `cuda-python` from `run:` entirely, because `cuda-python` has `run_exports` defined from version 11.7.1 onward.

Reply:
Good point. That should simplify things a fair bit

Marking this resolved

Reply:
Unresolving this -- I didn't actually _make_ that change of dropping cuda-python from `run:`. I want to save that for a follow-up PR if possible.

Reply:
Ah missed that initially. Tried to capture an updated set of changes below based on this ( https://github.com/rapidsai/rmm/pull/1223#pullrequestreview-1489515211 )

Reply:
I decided to go ahead and apply this change in 3213081. If it passes CI that's fine. Resolving.